trace-dispatcher/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for trace-dispatcher
 
+## 2.6.0
+
+* With a metrics prefix that can be set in the configuration (tcMetricsPrefix)
+  Metrics gets a type postfix (_int,_real, _counter)
```

Comment:
Just for context: these postfixes are alre part of the metrics names in legacy tracing?

Reply:
Partially, if you go ekg_direct it is own responsibility

Reply:
Thanks!